package.json

```
@@ -3,7 +3,7 @@
   "version": "0.0.1",
   "private": "true",
   "description": "Node schedule utilities for use with Renovate bot.",
-  "main": "index.js",
+  "main": "build/src/index.js",
```

```suggestion
  "main": "build/index.js",
```

re need to remove the additional `src` path